sql/moz-fx-data-shared-prod/stripe_external/customers_changelog_v1/metadata.yaml

```diff
@@ -0,0 +1,27 @@
+friendly_name: Stripe customers changelog
+description: |-
+  Changes to Stripe customers as recorded by Fivetran (https://fivetran.com/docs/applications/stripe).
+
+  IMPORTANT: This table should not be rebuilt from scratch because the underlying Fivetran table doesn't preserve historical changes.
+
+  This table's schema closely mirrors Stripe's customers API (https://stripe.com/docs/api/customers/object).
```

Comment:
Like [I mentioned for the subscriptions changelog](https://github.com/mozilla/bigquery-etl/pull/3950#discussion_r1231488835), I'm shaping the data to be like the API because I expect to replace Fivetran as the data source with either the SubPlat customers Firestore collection changelog sync or a custom sync of customer events from Stripe's API, both of which would have Stripe customer data in the API format.